src/main/java/org/opentripplanner/model/Station.java

```diff
@@ -159,13 +161,19 @@ public GeometryCollection getGeometry() {
   }
 
   private static GeometryCollection computeGeometry(WgsCoordinate coordinate, Set<StopLocation> childStops) {
-    var stationPoint =  getGeometryFactory().createPoint(coordinate.asJtsCoordinate());
-    var childGeometries = childStops.stream().map(StopLocation::getGeometry).collect(Collectors.toList());
-    childGeometries.add(stationPoint);
-
+    Point stationPoint = null;
+    var childGeometries = childStops.stream()
+                                                    .map(StopLocation::getGeometry)
```

Comment:
The indentation is a bit strange here. Is that intentional?

Reply:
fixed :)